test/support/fake_formatter.ex

```diff
@@ -6,15 +6,15 @@ defmodule Benchee.Test.FakeFormatter do
   end
 
   def write(output) do
-    send self(), {:write, output}
+    send(self(), {:write, output})
   end
 
   def output(suite) do
-    :ok = suite
-          |> format
-          |> write
+    :ok =
+      suite
+      |> format
+      |> write
```

Comment:
waiiit.... we can't write the other version anymore? I like it a lot when the functions aren't as long...

Reply:
Which other version? Like `:ok = suite |> format |> write`? I think that's ok, but since we already had it on multiple lines, it follows a rule that's essentially this:

```elixir
  result =
    "any expression"
    <> " that spans over multiple lines"
    <> " gets indentation like this"
    <> " where the result is above and 2 spaces de-dented from the expression"
```

Reply:
I also like:

```elixir
result = something
         |> other
         |> bam
```

no biggy though